src/System.Security.Cryptography.Xml/src/System.Security.Cryptography.Xml.csproj

```diff
@@ -90,5 +90,20 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This isn't necessary; the resx will automatically be pulled in if there's a file names Strings.resx